pull-report.js

```diff
@@ -171,9 +172,12 @@ if (require.main === module) {
   if (!program.org) {
     throw new Error("Must specify 1+ organization names");
   }
-  if (!(program.ghUser && program.ghPass)) {
-    throw new Error("Must specify GitHub user / pass in .gitconfig or " +
-      "on the command line");
+  // If we have a token, no need for user/password
+  if (!program.ghToken) {
+    if (!(program.ghUser && program.ghPass)) {
+      throw new Error("Must specify GitHub user / pass in .gitconfig or " +
+		      "on the command line");
```

Comment:
Trailing lines get indented at two per project style (no worries -- not documented :wink: )